### PR TITLE
Improve Typography

### PR DESCRIPTION
**Pull request by tushar5526**
Related to #371 
This PR covers
- `text_width`
- `text_leading`
- `text_align`
- Added support for multiline pythonic strings and multiline strings

### CODE REVIEW

**Comment by tushar5526** [comment]
@ziyaointl I bumped the `numpy` in this PR. Should I make a separate PR for that rebase this on the updated master ?

**Review by ziyaointl — COMMENTED** [review]
Great work! Yeah I do think making a separate PR for numpy then rebasing is the best practice.

File: p5/visualTests/sanityTests/tmp_test37.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from p5 import *
```
[the remark this answers is not in the document]
Comment (tushar5526) [reply p5/visualTests/sanityTests/tmp_test37.py:1]: Yes, it was not supposed to go there. Sanity testing scripts generate a temp file with renderer stubs for testing. Thanks

File: p5/sketch/Skia2DRenderer/renderer2d.py [hunk 2]
```diff
@@ -134,10 +135,28 @@ def initialize_renderer(self, canvas, paint, path):
...
-        # assert (typeface is not None), "should not be NULL"
-        # handle alignment manually
+    def render_text(self, texts, x, y):
+        '''
+        :param text: List of strings to be rendered
+        :type text: list
+        '''
+        if not texts: return
+
+        text, texts = texts[0], texts[1:]
```
[the remark this answers is not in the document]
Comment (tushar5526) [reply p5/sketch/Skia2DRenderer/renderer2d.py:145]: Thank you. I made it an iterative O(n).

**Review by ziyaointl — APPROVED** [review]

**Pull request merged**